exchanges.yml

```
@@ -386,20 +386,39 @@ components:
       additionalProperties: false
       description: Object containing information about an active exchange.
       properties:
-        exchangeId:
-          type: string
-          description: The URL that uniquely identifies the exchange.
-        ttl:
-          type: string
-          description: The time to live for the exchange.
-        nextStep:
-          type: string
-          description: The interaction the exchange is expecting to happen next.
-        completedSteps:
-          type: array
-          description: The steps completed by the exchange previously.
-          items:
-            type: string
+        exchange:
+          type: object
+          description: The active exchange.
+          properties:
+            exchangeId:
+              type: string
+              description: The URL that uniquely identifies the exchange.
+            ttl:
+              type: string
+              description: The time to live for the exchange.
+            nextStep:
+              type: string
+              description: The interaction the exchange is expecting to happen next.
+            completedSteps:
+              type: array
+              description: The steps completed by the exchange previously.
+              items:
+                type: string
```

Simplified to match what's in the exchange.yml:

```suggestion
            step:
              type: string
              description: The current step in the exchange.
```